assignments/prototypes.js

```diff
@@ -22,6 +29,15 @@
   * takeDamage() // prototype method -> returns the string '<object name> took damage.'
   * should inherit destroy() from GameObject's prototype
 */
+function CharacterStats(charAttributes) {
+  GameObjects.call(this, charAttributes);
+  this.healthPoints = charAttributes.healthPoints;
+  this.name = charAttributes.name;
+  this.damage = function() {
+    return `${this.name} took damage.`;
+  }
+}
+
 
```

Comment:
It doesn't appear that you passed the prototype inheritance. At least, I'm not seeing the code?